apps/mobile/src/features/approver/layouts/base-stx-tx-approver.layout.tsx

```diff
@@ -80,82 +81,92 @@ export function BaseStxTxApproverLayout({
   });
 
   return (
-    <ApproverWrapper>
-      <Approver>
-        <Approver.Container>
-          <Approver.Header
-            title={t({
-              id: 'approver.signTransaction.title',
-              message: 'Sign Transaction',
-            })}
+    <Approver requester={origin}>
+      <Approver.Container>
+        <Approver.Header
+          title={t({
+            id: 'approver.signTransaction.title',
+            message: 'Sign Transaction',
+          })}
+        />
+        <Approver.Section>
+          <ApproverAccountCard
+            accounts={accounts.filter(
+              acc => makeAccountIdentifer(acc.fingerprint, acc.accountIndex) === accountId
+            )}
           />
-          <Approver.Section>
-            <ApproverAccountCard
-              accounts={accounts.filter(
-                acc => makeAccountIdentifer(acc.fingerprint, acc.accountIndex) === accountId
-              )}
-            />
-          </Approver.Section>
-          {isContractCall(tx.payload) && <ContractCallSummarySection txHex={txHex} />}
-          {isContractDeploy(tx.payload) && <ContractDeploySummarySection txHex={txHex} />}
-          {isTokenTransferPayload(tx.payload) && (
-            <Approver.Section>
+        </Approver.Section>
+        {isContractCall(tx.payload) && <ContractCallSummarySection txHex={txHex} />}
```

Comment:
`@stacks/transactions` does have methods for `isTokenTransferPayload`, `isContractCallPayload`, and `isSmartContractPayload` if helpful to replace these?

Reply:
Thanks Fara. This isn't new code, but tagging @edgarkhanzadian.